scripts/debug

```diff
@@ -0,0 +1,183 @@
+# 3. Neither the name of the University nor the names of its contributors
+#    may be used to endorse or promote products derived from this software
+#    without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR 
+# A PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE HOLDERS OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, 
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, 
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR 
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF 
+# LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING 
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS 
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+# Try to load Umbrel OS version info to detect Umbrel OS
+[[ -f "/etc/default/umbrel" ]] && source "/etc/default/umbrel"
+
+# function to upload the output to hastebin
+# Source: https://github.com/seejohnrun/haste-client#lightweight-alternative
+haste() { a=$(cat); curl -X POST -s -d "$a" https://hastebin.com/documents | awk -F '"' '{print "https://hastebin.com/"$4}'; }
+
+echo "====================="
+echo "= Umbrel debug info ="
+echo "====================="
+
+UMBREL_ROOT="$(dirname $(readlink -f "${BASH_SOURCE[0]}"))/.."
+cd "${UMBREL_ROOT}"
+
+echo
+echo "Umbrel version"
+echo "--------------"
+cat info.json | jq .version | sed 's/"//g'
+
+if [[ ! -z "${UMBREL_OS:-}" ]]; then
+    echo
+    echo "Umbrel OS version"
+    echo "-----------------"
+    echo $UMBREL_OS
+
+    echo
+    echo "Raspberry Pi Model"
+    echo "------------------"
+
+    cat /proc/cpuinfo | tail -3
+
+    echo
+    echo "Firmware"
+    echo "--------"
+
+    vcgencmd version
+
+    echo
+    echo "Temperature"
+    echo "-----------"
+
+    vcgencmd measure_temp
+
+    echo
+    echo "Throttling"
+    echo "----------"
+
+    vcgencmd get_throttled
+
+    echo
+    echo "Startup service logs"
+    echo "--------------------"
+    journalctl --unit=umbrel-startup.service | tail -n 30  | sed '/onion/d'
+
+    echo
+    echo "External storage service logs"
+    echo "-----------------------------"
+    journalctl --unit=umbrel-external-storage.service | tail -n 30
+fi
+echo
+echo "Filesystem information"
+echo "----------------------"
+
+df -h
+echo
+cat /proc/swaps
+
+
+echo
+echo "Networking information"
+echo "----------------------"
+echo
+
+ifconfig | sed -e "s/\([0-9a-fA-F]\{1,4\}:\)\{7,7\}[0-9a-fA-F]\{1,4\}/y.y.y.y.y.y.y.y/g" | sed -e "s/[0-9a-fA-F]\{1,4\}:\(:[0-9a-fA-F]\{1,4\}\)\{1,4\}/y::y.y.y.y/g" | sed -e "s/\([0-9]\{1,3\}\.\)\{3,3\}[0-9]\{1,3\}/x.x.x.x/g" | sed -e "s/\([0-9a-fA-F]\{2,2\}\:\)\{5,5\}[0-9a-fA-F]\{2,2\}/m.m.m.m/g"
+
```

Comment:
We can probably get rid of the networking info as it takes too much space in the logs and we never actually had the need to look at it and instead let's tail `karen's` logs as those are quite useful in the event of a failed update, backup, app installs/uninstalls, etc.

```
tail -n 50 logs/karen.log | sed '/onion/d'
```